randovania/games/fusion/gui/ui_files/games_tab_fusion_widget.ui

```diff
@@ -136,17 +136,54 @@
+           <string>Randovania and the MARS patcher make some changes to the original game in order to improve the game experience or to simply fix bugs in the original game. Many changes are optional and can be disabled in the options Randovania provides, but the following are always there: 
+
+### Gameplay
+
+- The Security Rooms are now item locations and the locks can be shuffled into the item pool. 
```

Comment:
honestly i would probably remove this point
That, or rephrase it into "the event system from the game has been removed" or something. Then we can also mention there that the sa-x will never patrol, emergency in :dog:, the reactor shutdown and the secret lab visit will never happen. Maybe also mention that zazabi => cocoons, nettori => kihunters?